app/src/main/java/org/sopt/dosopttemplate/presentation/auth/SignUpActivity.kt

```diff
@@ -5,13 +5,15 @@ import android.os.Bundle
 import android.view.MotionEvent
 import android.view.View
 import androidx.activity.viewModels
+import dagger.hilt.android.AndroidEntryPoint
 import org.sopt.dosopttemplate.R
 import org.sopt.dosopttemplate.databinding.ActivitySignUpBinding
 import org.sopt.dosopttemplate.presentation.model.UserInfo
 import org.sopt.dosopttemplate.util.binding.BindingActivity
 import org.sopt.dosopttemplate.util.extension.hideKeyboard
 import org.sopt.dosopttemplate.util.extension.showSnackbar
 
+@AndroidEntryPoint
 class SignUpActivity : BindingActivity<ActivitySignUpBinding>(R.layout.activity_sign_up) {
     private val viewModel: AuthViewModel by viewModels()
```

Comment:
```suggestion
 private val viewModel by viewModels<AuthViewModel>()
```
https://developer.android.com/kotlin/ktx/extensions-list?hl=ko
ktx에서 제공해주는 extension을 이용하면 해당 액티비티의 생명주기에 맞게 뷰모델의 생명주기도 파괴되기 때문에 더더욱 메모리 리소스를 줄일 수 있습니다